client/app/scripts/superdesk-authoring/metadata/views/metadata-widget.html

```diff
@@ -88,10 +88,20 @@
 			<label translate>Genre</label>
 			<div sd-meta-terms data-item="item" data-field="genre" data-unique="value" data-list="metadata.genre" ng-disabled="!_editable" data-change="autosave(item)" data-postprocessing="processGenre()"></div>
 		</li>
-		<li>
-			<label translate>Restrictions</label>
-			<div class="data visible" ng-if="metadata.geographical_restrictions">
-				<div sd-meta-dropdown data-list="metadata.geographical_restrictions" data-item="item" data-field="restrictions" data-change="autosave(item)" ng-disabled="!_editable"></div>
+		<li class="terms-box">
+			<label translate>Targeted For</label>
+			<div ng-if="metadata.targeted_for">
+				<span sd-check ng-model="item.negation" tooltip="Adds Negation" tooltip-placement="right" ng-disabled="!_editable"></span>
+				<select id="targeted_for_list" ng-model="item.targeted_for_value" ng-disabled="!_editable">
+					<option value=""></option>
+					<option value="{{g.value}}" ng-repeat="g in metadata.targeted_for track by g.name">{{:: g.name}}</option>
+				</select>
+				<button class="btn btn-small btn-primary" type="button" ng-click="addTargeted()" ng-disabled="!_editable" translate>Add</button>
+				<div class="terms" ng-if="item['targeted_for'].length">
+					<ul>
+						<li ng-repeat="t in item['targeted_for']" ng-click="removeTargeted(t)">{{ :: t['allow'] == false ? 'Not ' + t.name : t.name}}<i class="icon-close-small"></i></li>
```

Comment:
Based on the rest of the code, shouldn't we be using the identity comparison (`===`) here as well?